### PR TITLE
HCP Connectome Workbench colormaps

### PR DESCRIPTION
**Pull request by dangom**
This PR adds support for two colormaps that have been gaining popularity with the HCP Connectome Workbench, namely _roy_big_bl_ and _videen_style_. 

Examples:

videen

roy_big_bl

No comments whether they are good colormaps at all, but potentially relevant for those working with HCP data to plot and compare results with the literature.

### CODE REVIEW

File: doc/plotting/index.rst [hunk 1]
```diff
@@ -231,6 +231,32 @@ Different display modes
 
 ================= =========================================================
 
+Available Colormaps
+========================
+
+Nilearn's plotting library ships with a set of extra colormaps, as seen in the
+image below::
```
Comment (GaelVaroquaux) [inline doc/plotting/index.rst]: This should be put in a example, ie a Python file in examples/plottting/plot_colormaps.py. These get executed by our build system when you build the docs.

**Comment by dangom** [comment]
Thanks for the comments. These colorbars are misleading just like the old rainbow color map, but unfortunately they've been gaining quite some popularity as the default of the Workbench.

Regarding the collage, I've been working on [some code that does that](https://github.com/dangom/nilearn-cli/blob/master/nilearn_cli/surface_plot.py#L70) recently.  I'd be happy to eventually add it into nilearn. 

One possibility would be to have an extra option for the "view" kwarg of plot_surf_stat_map? Something in the lines of:

    plotting.plot_surf_stat_map(surface, texture_l,  view='all', **kwargs)

The problem with this approach is that the collage code needs (in my case) 4 axes instead of only 1, and it needs multiple surfaces and textures. It feels like the calling signature would change so much that a separate function would be best.

Perhaps something like:

    plotting.plot_stat_on_surf(stat_map_img, bg_mesh=<fsaverage>, **kwargs)

Not so sure about the name, though.

**Comment by bthirion** [comment]
```plotting.plot_sur_stat_montage(stat_map_img, bg_mesh=<fsaverage>, **kwargs)``` ?

**Comment by GaelVaroquaux** [comment]
plotting.plot_surf_stat_montage feels right to me. Let's hear what others
think.

**Comment by jeromedockes** [comment]
I like `plot_surf_stat_montage` because montage makes it clear there are
multiple views. however, if I understand correctly, a big difference with
`plot_surf_stat_map` is that this function accepts a volume image and projects
it, whereas `plot_surf_stat_map` takes a texture. for consistency with
`view_img_on_surf`, maybe this could be called `plot_stat_map_on_surf`, or
`plot_stat_map_on_surf_montage`. note that many functions, like `plot_stat_map`
or `plot_glass_brain`, accept a `display_mode` argument and display multiple
views.

**Comment by GaelVaroquaux** [comment]
> accept a display_mode argument and display multiple views.

+1 for display_mode, but we need to work out the semantics.

**Comment by GaelVaroquaux** [comment]
Merged as part of #1720. Thanks!

**Pull request closed without merging**